### DISCUSSION TITLE
협업툴 [JANDI] 활용

### DISCUSSION THREAD

**Discussion by dlskawns96** [TeamOffOff/OffOff#13] [Q&A] [answered]
# JANDI
https://www.jandi.com/landing/kr
협업용 메신저

## 주요 기능
- 깃허브 이벤트 (이슈, PR, Push, ...)알림
- 구글 캘린더 연동
- 회의록
- 채팅
- 기타 등등

## 다운로드
PC: https://www.jandi.com/landing/kr/download
안드로이드: https://play.google.com/store/apps/details?id=com.tosslab.jandi.app
iOS: https://apps.apple.com/kr/app/jandi/id904895208

## 스크린샷
<img width="500" src="https://user-images.githubusercontent.com/22260098/126055080-2b93673d-2cf6-4acb-a055-a9744126be32.png"> <img width="500" src="https://user-images.githubusercontent.com/22260098/126055082-6729d483-3ff0-4f1a-941d-2394c8d43555.png">

## 투표
사용해보자 ->    👍 
카톡이나쓰자 -> 👎

**Comment by dlskawns96** [comment, marked as answer]
👍

**Comment by yuuuzzzin** [comment]
👍👍

**Comment by YouminYoumin** [comment]
👍
